css/global.scss

```diff
@@ -800,7 +805,7 @@ text-align: center !important
         min-width: 250px;
         max-width: 250px;
     }
-    .sezzle-hiw-pie-bg .breakdown-row span {
+    .sezzle-hiw-pie-bg .breakdown-row .due {
```

Comment:
I couldn't find the classname `.due` being referred anywhere in this change. Where is it referred?